source/blog/articles/2015-02-26-london.html.md

```diff
@@ -15,7 +15,7 @@ READMORE
 
 ### Friends
 
-A good friend and ex-coworker moved there last September. He has invited me several times to come see him. So I taught it was logic to begin my journey by actually visiting him.
+[Kevin](https://kevin.disneur.me/), a good friend and ex-coworker moved there last September. He has invited me several times to come see him. So I taught it was logic to begin my journey by actually visiting him.
```

Comment:
Add some SEO juice for friends 

[Kevin, a ruby on rails developer](https://kevin.disneur.me/), good friend and ex-coworker moved there last September.